_posts/2021-01-14-macpatch-3.5.0.5.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "MacPatch 3.5.0.5 released"
+categories: release
+---
+
+MacPatch -- used at LLNL to manage 3,000+ computers -- simplifies the act of patching and installing software on Mac OS X based systems. The client relies on using the built-in software update application for patching the Mac OS X system updates and its own scan and patch engine for custom patches. v3.5.0.5 is a major release with a agent installation tracking, dynamic patch list, updated MPAgentUploader, and more.
```

Comment:
Technically, I think the proper spelling now adays is "macOS" instead of "Mac OS X"

Reply:
committed Mac OS X --> macOS in both the new post and an older one about MacPatch